client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCompleteOrderForm.tsx

```diff
@@ -29,35 +32,39 @@ export interface CheckoutFormInfo {
   visitSource?: string;
   seatingAcc: string;
   comments?: string;
+  donation?: number;
 }
 
 /**
  * Used to complete order
  *
  * @param {Function} onSubmit
- * @param {Function} onBack
  * @param {boolean} disabled
  * @param {boolean} donationForm
  */
 type CompleteOrderFormProps = {
   onSubmit: (formData: CheckoutFormInfo) => any;
-  onBack: () => any;
+  backButtonRoute: string;
+  eventDataFromPurchase: EventRow[];
   disabled: boolean;
   donationForm?: boolean;
 };
 /**
  * Displays the complete order form
  *
  * @param {Function} onSubmit onSubmit callback function
- * @param {Function} onBack onBack callback function
+ * @param CompleteOrderFormProps.backButtonRoute
+ * @param CompleteOrderFormProps.eventDataFromPurchase
  * @returns {ReactElement}
  */
-export default function CompleteOrderForm({
+export default function AdminCompleteOrderForm({
   onSubmit,
-  onBack,
+  backButtonRoute,
+  eventDataFromPurchase,
   disabled,
   donationForm,
```

Comment:
Since these props don't do anything, they can be removed

Reply:
removed